django/db/models/aggregates.py

```diff
@@ -115,8 +118,8 @@ def __init__(self, expression, filter=None, **extra):
             raise ValueError('Star cannot be used with filter. Please specify a field.')
         super().__init__(expression, filter=filter, **extra)
 
-    def convert_value(self, value, expression, connection):
-        return 0 if value is None else value
+    def get_empty_result(self):
+        return 0
```

Comment:
I suspect you also want to do this for `django.contrib.postgres.aggregates.RegrCount()`?